src/engines/Cesium/SpatialId/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { ClassificationType, Color, HeightReference, Rectangle } from "cesium";
+import { FC, memo, useMemo } from "react";
+import { Entity } from "resium";
+
+import { SpatialIdSpaceType } from "../../../Map/SpatialId/types";
+
+const DEFAULT_COLOR = "#00bebe";
+
+type SpatialIdComponentProps = {
+  space: SpatialIdSpaceType;
+  color?: string;
+};
+
+const SpatialIdSpace: FC<SpatialIdComponentProps> = memo(({ space }) => {
+  const options = useMemo(
+    () => ({
+      rectangle: {
+        coordinates: Rectangle.fromDegrees(...space.wsen),
+        height: space.type === "coordinate" ? undefined : space.height,
+        heightReference: space.type === "coordinate" ? undefined : HeightReference.NONE,
+        extrudedHeight: space.type === "coordinate" ? undefined : space.extrudedHeight,
+        clampToGround: space.type === "coordinate",
+        fill: true,
+        outline: space.type !== "coordinate",
+        outlineColor: Color.fromCssColorString(space.color ?? DEFAULT_COLOR).withAlpha(
+          space.type === "floor" ? 0.5 : 1,
+        ),
+        material: Color.fromCssColorString(space.color ?? DEFAULT_COLOR).withAlpha(
+          space.type === "floor" ? 0.05 : space.type === "coordinate" ? 0.2 : 0.2,
+        ),
```

Comment:
Can't we use `space.colorFloor` and `space.colorCoordinate`? I thought it might be easy to understand and be customizable.

Reply:
I mean we should use `space.floor_color` and `space.coorinate_color`, then we don't need to check these condition. Also it allows user change the color depends on `space.type`.

Reply:
As discussed on slack i'll keep it as-is for now. I'll refactor it if we have the request to define the color independently.